src/istream.cpp

```diff
@@ -35,20 +36,17 @@ auto write_data_object(rodsEnv& env, const po::variables_map& vm, io::client::de
 
 int main(int argc, char* argv[])
 {
-    if (argc == 1) {
-        std::cerr << "Error: Invalid number of arguments.\n";
-        return 1;
-    }
-
     po::options_description desc{"Options are"};
     desc.add_options()
-        ("help,h", "Produce this message.")
-        ("offset,o", po::value<int_type>()->default_value(0), "The position of the read/write pointer.")
-        ("count,c", po::value<int_type>()->default_value(all_bytes), "The number of bytes to read/write.")
-        ("no-trunc", po::bool_switch(), "Do not truncate the data object on write.")
-        ("append,a", po::bool_switch(), "Append to the data object on write.")
-        ("stream_operation", po::value<std::string>(), "The stream operation (e.g. read or write).")
-        ("logical_path", po::value<std::string>(), "The logical path of a data object.");
+        ("help,h", "")
+        ("R,R", po::value<std::string>(), "")
```

Comment:
R,R ?   resource,R instead?

Reply:
Boost program options does not let you specify only short names. To get around that, you have to give it two short names. Essentially, this means the program treats -R and --R the same.

There may be a hook for controlling this behavior better, but this was the solution I found online.

Reply:
The --R is not documented in the help text either.

Thoughts?

Reply:
what you've got below seems fine...

`The root resource to read from or write to.`

Reply:
happy to have a `--resource,-R` as long/short option... always good to nudge people to use all the letters.

Reply:
Ok. I'll add it. It makes the code more readable too.